pkgs/by-name/qu/quill-cpp/package.nix

```diff
@@ -0,0 +1,33 @@
+{
+  cmake,
+  fetchFromGitHub,
+  lib,
+  stdenv
+}:
+
+stdenv.mkDerivation rec {
+  pname = "quill-cpp";
+  version = "3.8.0";
+
+  src = fetchFromGitHub {
+    owner = "odygrd";
+    repo = "quill";
+    rev = "v${version}";
+    sha256 = "sha256-8AS/1Yb8sHO/0wpjNZk/b18QtDM2tR7+DFnClFeMMPI=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+  ];
+
+  meta = with lib; {
+    description = "A high-performance, cross-platform logging library";
+    homepage = "https://github.com/odygrd/quill";
+    license = licenses.mit;
+    maintainers = [ maintainers.vaci ];
+    sourceProvenance = [
```

Comment:
Not needed

Reply:
Can any/of these metadata be omitted, then?

Reply:
> The presence of a particular sourceType in a package’s meta.sourceProvenance list indicates that the package contains some components falling into that category, though the absence of that sourceType does not guarantee the absence of that category of sourceType in the package’s contents. A package with no meta.sourceProvenance set implies it has no known sourceTypes other than fromSource.

In https://nixos.org/manual/nixpkgs/unstable/#lib.sourceTypes.fromSource

Reply:
I see, thanks for the explanation!